ports/zephyr/prj_mimxrt1050_evk.conf

```diff
@@ -1,3 +1,9 @@
 # Required for zephyr.DiskAccess block devices
 CONFIG_DISK_ACCESS=y
 CONFIG_DISK_ACCESS_SDHC=y
+
+CONFIG_USB=y
+CONFIG_USB_DEVICE_STACK=y
+CONFIG_USB_DEVICE_PRODUCT="Zephyr Micropython"
```

Comment:
please spell it "MicroPython" with an uppercase "P"

Reply:
Done